Dockerfile

```diff
@@ -5,6 +5,14 @@
 
 # Pull base image.
 FROM dockerfile/ubuntu
+
+# Set the env variables to non-interactive
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
Hi Codewars/codewars-runner,
A message from Matthew Wampler-Doty was left on IQ Elite for you. 
Click here a.iqelite.com/j/0JmRlo/o8n/?hg=PoO4syC8qA to read your message 


The IQ Elite Team



---------------------------------------------------------------------------
This message was meant for reply@reply.github.com. If you do not want to continue receiving such mails from IQ Elite, please click on this link: a.iqelite.com/common/unsubscribe/?code=92b6bb31&email=reply@reply.github.com&hg=PoO4syC8qA
IQ Elite is a service with main office: C/ Gran De Gràcia 15, 1a 1a, 08012 Barcelona, Spain.
---------------------------------------------------------------------------